agent/runner/jobs/pbm_helpers.go

```diff
@@ -338,7 +339,12 @@ func waitForPBMRestore(ctx context.Context, l logrus.FieldLogger, dbURL *url.URL
 				err = execPBMCommand(ctx, dbURL, &info, "describe-restore", name)
 			}
 			if err != nil {
-				return errors.Wrap(err, "failed to get restore status")
+				if maxRetryCount > 0 {
+					maxRetryCount--
+					l.Errorf("PMM failed to get backup restore status and will retry: %s", err)
```

Comment:
```suggestion
					l.Warnf("PMM failed to get backup restore status and will retry: %s", err)
					continue
```

Reply:
Also I think we should reset `maxRetryCount` in case when we got correct response.